ctez.mligo

```diff
@@ -97,6 +81,43 @@ let get_oven_delegate (oven_address : address) : (key_hash option) contract =
   | None -> (failwith error_OVEN_MISSING_DELEGATE_ENTRYPOINT : (key_hash option) contract)
   | Some c -> c
 
+let get_ctez_mint_or_burn (fa12_address : address) : (int * address) contract =
+  match (Tezos.get_entrypoint_opt  "%mintOrBurn"  fa12_address : ((int * address) contract) option) with
+  | None -> (failwith error_CTEZ_FA12_CONTRACT_MISSING_MINT_OR_BURN_ENTRYPOINT : (int * address) contract)
+  | Some c -> c
+
+// Returns the price dy/dx, i.e. a map by multiplication ∆x => ∆y, at a given point (x,y)
+let price_cash_to_token (target : nat * nat) (cash : nat) (token : nat) : nat = 
+    let (a,b) = target in 
```

Comment:
Intuitively this logic belongs in the CFMM who should export prices as fractions to the ctez contract, not the ctez contract itself.

Reply:
Price is stored as a `nat` in the code currently—that's how it is in the ctez repo so I kept that as default. Should it be stored as a fraction?